### ISSUE TITLE
Is it an issue that nbdev_build_docs does not run all imports?

### ISSUE THREAD

**Issue by pete88b** [fastai/nbdev#171]
Please see https://github.com/pete88b/decision_tree/blob/master/71_tensor_patch.ipynb

I think we have a few choices
- leave as-is
- update `ExecuteShowDocPreprocessor` to run all imports
    - not complete cells, but any line starting with `from` or `import`
- add a flag to settings.ini to opt-in to new "run all imports" behavior

happy to make a PR either way

**Comment by sgugger** [comment]
Are you sure you have exported the notebook before running this? Since 
```
from fastai2.torch_basics import *
from nbdev.showdoc import *
```
should have been exported, it should be in the namespace when the notebook is executed since it imports the corresponding module.

**Comment by pete88b** [comment]
Congratulations on the new job (o:

for sure - this has been exported - http://localhost:8888/edit/decision_tree/decision_tree/test/tensor_patch.py
I can also see the source of `_import_show_doc_cell`
```
from nbdev.showdoc import show_doc
from decision_tree.test.tensor_patch import *
```
but ... importing a module doesn't seem to run its imports
![image](https://user-images.githubusercontent.com/11458288/83637665-df1fae80-a59f-11ea-85f9-2d2202a540cb.png)

I was wondering if this should pick up all imports?
`_re_cell_to_execute = ReLibName(r"^\s*show_doc\(([^\)]*)\)|^from LIB_NAME\.", re.MULTILINE)`

**Comment by sgugger** [comment]
Weird. I guess we could change to detect all imports and execute them.

**Comment by pete88b** [comment]
cool - i'll pop a PR in soon

**Comment by sgugger** [comment]
Closed by #173
